content/projects/megaresume/_index.md

```diff
@@ -3,6 +3,11 @@ title: MegaResume
 summary: "A resume building application"
 startDate: 2020-01
 endDate: 2020-11
+tags:
```

Comment:
I would like to use the contents of the front matter for each of these pages to populate the partials in the home page, but I'm still trying to figure out how to filter out the contents accordingly.